src/git/commit_filter.go

```diff
@@ -79,7 +119,7 @@ func (s *CommitFilter) Commits(lastHash string) ([]Commit, error) {
 
 // commitChangesIncluded returns true if at least one of the changes is included in includedDirs.
 func (s *CommitFilter) commitChangesIncluded(files []string) bool {
-	if len(s.includedDirs) < 1 {
+	if len(s.includedDirs) < 1 && len(s.includedFiles) < 1 {
```

Comment:
Nit: we may want to update the function commit to include `includedFiles` or just make the comment more generic.